package.json

```diff
@@ -301,7 +301,7 @@
     "@metamask/eth-query": "^4.0.0",
     "@metamask/eth-sig-util": "^7.0.1",
     "@metamask/eth-snap-keyring": "^5.0.1",
-    "@metamask/eth-token-tracker": "^9.0.0",
+    "@metamask/eth-token-tracker": "^10.0.1",
```

Comment:
This version bump was necessary for testing with `StreamProvider` (which, unfortunately, only supports `EIP-1193`). However, since we are currently using `MetaMaskInpageProvider`, this bump can be reverted if it is not relevant to this PR.